src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -25,12 +26,16 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             placeholder="Message prompt"
             keyboardAppearDelay={500}
             editable={!props.isOtherProfile}
+            maxLength={350}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a `maxLength` prop to the `AutoFocusTextArea` component is a good practice to prevent users from entering more characters than the system can handle. However, ensure that the backend also has similar validation for the bio length to maintain data consistency.
```diff
+ maxLength={350}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->